python/servo/testing_commands.py

```diff
@@ -414,6 +414,22 @@ def test_dromaeo(self, tests, servo_binary: str, bmf_output: str | None = None):
     def test_speedometer(self, servo_binary: str, bmf_output: str | None = None):
         return self.speedometer_runner(servo_binary, bmf_output)
 
+    @Command('test-size', description="Prints servo's size", category='testing')
+    @CommandArgument('--bmf-output', default=None, help="Specify BMF JSON output file")
+    @CommandBase.common_command_arguments(binary_selection=True)
+    def test_size(self, servo_binary: str, bmf_output: str | None = None):
+        size = os.path.getsize(servo_binary)
+        print(size)
+        if bmf_output:
+            with open(bmf_output, 'w', encoding='utf-8') as f:
+                json.dump({
+                    'servo': {
+                        'file-size': {
+                            'value': float(size),
+                        }
+                    }
+                }, f, indent=4)
+
```

Comment:
Is the BMF file format defined somewhere?

Reply:
https://bencher.dev/docs/reference/bencher-metric-format/